cmd/migrate.go

```diff
@@ -147,7 +147,14 @@ func migrateReset() error {
 
 func initMigrations(vipr *viper.Viper, logger log.Logger) (*migrate.Migrate, error) {
 	pgCfg := flags.NewPostgresConfig(vipr)
-	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?", pgCfg.User, pgCfg.Password, pgCfg.Host, pgCfg.Port, pgCfg.Database)
+	var userPass string
```

Comment:
I would name this one userCredentials

Reply:
They are not always credentials since you might only have username in it